README.md

```diff
@@ -51,6 +55,60 @@ They can be rendered on both `Component` and `Group` entity pages.
 | `<EntityTimeToRecoveryCard />`      | Line chart showing Time to Recovery for the service.                  |
 | `<EntityTopContributorsTable />`    | Table showing top contributors by pull request count for the service. |
 
+## DX Service Cloud Components
+
+These components visualize Scorecards and Tasks for an entity.
+
+| Component                  | Description                                                               |
+| -------------------------- | ------------------------------------------------------------------------- |
+| `<EntityScorecardsPage />` | Dashboard showing scorecard details for the service.                      |
+| `<EntityTasksPage />`      | Dashboard showing outstanding tasks for the service, grouped by priority. |
+| `<EntityScorecardsCard />` | Info card showing current scorecard levels and checks for the service.    |
+| `<EntityTasksCard />`      | Info card showing outstanding tasks for the service.                      |
+
+These components expect entities to have the same identifer in Backstage and DX. For example, if you have a Backstage entity with the `metadata.name` of `my-app`, then you must also have an entity in the DX Catalog with the identifier of `my-app` so the info cards can fetch the data successfully.
+
+Install the full-page components by defining routes in the service entity page:
+
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import { EntityScorecardsPage, EntityTasksPage } from '@get-dx/backstage-plugin';
+
+const serviceEntityPage = (
+  <EntityLayout>
+    {/* ... */}
+
+    <EntityLayout.Route path="/dx-scorecards" title="Scorecards">
+      <EntityScorecardsPage />
+    </EntityLayout.Route>
+
+    <EntityLayout.Route path="/dx-tasks" title="Tasks">
+      <EntityTasksPage />
+    </EntityLayout.Route>
+
+    {/* ... */}
+  </EntityLayout>
+)
+```
+
+### Additional setup
```

Comment:
Do we also want to add this content to the [backstage-backend-plugin README](https://github.com/get-dx/backstage-backend-plugin)?

Reply:
The proxy parts? Idk. I don't think they need the backend-plugin for the proxy to work?

Reply:
Oh you're right. The _instance backend_ needs it configured but the `backstage-backend-plugin` doesn't actually need it. I was conflating the two. Cool, I won't worry about updating that repo then.